ui/components/app/confirm/info/row/row.tsx

```diff
@@ -16,7 +16,7 @@ import {
   TextVariant,
 } from '../../../../../helpers/constants/design-system';
 
-export enum ConfirmInfoRowVariant {
+export enum ConfirmInfoRowState {
```

Comment:
I'm not sure I fully understand this name change.

This was intended to be for the different UI variants of the row components. What other kind of usage of `variant` would be expected here? 🤔

Reply:
I thought there could be confusion between "type" and "variant" while I was creating a "ConfirmInfoRowType" variable to represent e.g. address or value-double.

Then I thought "state" would be a more explicit representation here. I didn't realize it would be a breaking change for the Snaps API 🤔

I don't have any examples at this time about variants of variant. 

maybe it would be better to keep it as is 

@FrederikBolding